src/js/utils/getHexFromNumber.js

```diff
@@ -1,5 +1,11 @@
-export default number => {
-    const hex = number.toString(16);
+export default ( number, withPrefix = true, withSpace = false ) => {
+    let hex = number.toString(16);
 
-    return `0x${hex.length % 2 === 0 ? hex : `0${hex}`}`;
+    hex = hex.length % 2 === 0 ? hex : `0${hex}`;
+
+    if ( withSpace ) {
+        hex = hex.match(/.{2}/g).join(' ');
```

Comment:
we can get here something like this `0x07 5b c7 c8`

Reply:
fixed as discussed